subtrack.MAUI/Pages/UpsertSubscription.razor

```diff
@@ -33,6 +33,18 @@
     <div class="form-group" style="padding-top: 10px;">
         <InputTextArea class="form-control" rows="3" placeholder="Description" @bind-Value="@_subscription.Description"></InputTextArea>
     </div>
+    <div style="display: flex;padding-top: 10px;">
```

Comment:
interval and occurrence should be on the same line as showed in the mockups.

**Alt 1:** Paid every {interval} {occurrence}
**Alt 2:** Every {interval} {occurrence}
**Alt 3:**
Billing Cycle
every {interval} {occurrence}
**Alt 4:**
Billing
every {interval} {occurrence}